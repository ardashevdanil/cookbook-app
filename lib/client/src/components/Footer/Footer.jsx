import React from 'react';
import './Footer.css';

import NavBar from '../../containers/NavBar/NavBar';

function Footer() {
  return (
    <div className="Footer">
      <div className="Footer__logo" />
      <NavBar />
      <div className="Footer__copyright">
              &copy; 2016-2017 LEMON ALL RIGHT RESERVED
      </div>
    </div>
  );
}

export default Footer;
