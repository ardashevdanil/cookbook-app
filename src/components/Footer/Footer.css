.Footer {
	height: 182px;
  background-color: #2f2f2f;
  color: #fefefe;
  padding: 0 55px 0 86px;
	display: flex;
	align-items: center;
  justify-content: space-between;
  box-shadow: 0 0 13px 0 rgba(11, 12, 12, 0.15);
}

.Footer__logo {
  height: 36px;
	width: 87px;
  background: url('./logoDark.png');
  background-size: cover;
}

.Footer__copyright {
  font-size: 12.3px;
}

@media only screen and (max-width: 1120px) {
  .Footer {
    height: auto;
    padding: 11.1vw 6.9vw;
    box-shadow: none;
    justify-content: space-between;
    align-items: stretch;
    flex-wrap: wrap;
  }

  .Footer__logo {
    height: 12.5vw;
    width: 30.8vw;
  }

  .Footer__copyright {
    font-size: 4vw;
    padding-top: 5.6vw;
    padding-left: 42vw;
    border-top: 0.63vw solid #fefefe;
    text-align: right;
  }
}