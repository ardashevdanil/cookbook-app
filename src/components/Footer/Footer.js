import React from 'react';
import './Footer.css';

import NavBar from '../NavBar/NavBar';

function Footer(props) {
	return(
		<div className='Footer'>
			<div className='Footer__logo'></div>
			<NavBar />
      <div className='Footer__copyright'>
        &copy; 2016-2017 LEMON ALL RIGHT RESERVED
      </div>
		</div>
	);
}

export default Footer;