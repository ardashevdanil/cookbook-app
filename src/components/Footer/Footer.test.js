import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

import Footer from './Footer';

Enzyme.configure({ adapter: new Adapter() });

describe('test Footer component', () => {
	const component = Enzyme.shallow(
		<Footer />
	);

	test('renders correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
})