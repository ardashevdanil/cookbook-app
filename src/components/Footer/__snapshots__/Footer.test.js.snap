// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Footer component renders correctly 1`] = `
"<div className=\\"Footer\\">
  <div className=\\"Footer__logo\\" />
  <NavBar />
  <div className=\\"Footer__copyright\\">
    Â© 2016-2017 LEMON ALL RIGHT RESERVED
  </div>
</div>"
`;
